src/extra/AutoUIApp/Sidebar.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import styled from 'styled-components';
+import { useLocation } from 'react-router-dom';
+import { OpenApiJson } from './openApiJson';
+import { Flex, FlexProps } from '../../components/Flex';
+import { Txt } from '../../components/Txt';
+
+const SidebarWrapper = styled(Flex)`
+	height: calc(100vh - 60px);
```

Comment:
Can you please add a comment here noting that the 60px is the header height? So that it doesn't feel like we're just randomly subtracting 60px